dev/NumberBox/NumberBox.cpp

```diff
@@ -0,0 +1,312 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "common.h"
+#include "NumberBox.h"
+#include "RuntimeProfiler.h"
+#include "ResourceAccessor.h"
+
+using namespace std;
+
```

Comment:
Don't do "using namespace std", this codebase always uses explicit std:: on types.